File: crates/axhub-helpers/tests/fixtures/ast-validate/node/good.ts
import { db } from "@axhub/sdk";

// owner-scoped 테이블은 무필터 list/count 가 정당해요 (advisory warn 은 떠도 block 아님).
export async function loadPosts(ownerId: string) {
  const mine = await db.table("posts").list();
  const total = await db.table("posts").count();
  const filtered = await db.table("posts").eq("owner_id", ownerId).limit(20).list();
  return [mine, total, filtered];
}

File: crates/axhub-helpers/tests/fixtures/ast-validate/python/good.py
from axhub import db


def load_posts(client, owner_id):
    # owner-scoped 테이블은 무필터 list/count 가 정당해요.
    mine = db.table("posts").list()
    total = db.table("posts").count()
    filtered = db.table("posts").eq("owner_id", owner_id).limit(20).list()
    return mine, total, filtered
